src/input/index.js

```diff
@@ -47,6 +48,7 @@ class Input extends PureComponent {
         <li className={cx('tag-item')}>
           <input
             type="text"
+            disabled={this.props.disabled}
```

Comment:
Can you move it to the props destructuring statement at the beginning - to keep things consistent?

```
const { ..., disabled, ...} = this.props
...
disabled={disabled}
...
```